t04_refactored.py

```diff
@@ -14,7 +14,7 @@
 # This new version will take advantage of functions, as well as
 # demonstrate the value of git as a tool for collaborating.
 # https://docs.google.com/document/d/12AqOblfS_IRP24dKtyJrNh9a8E8JcmrvfzseTkbixyU/edit#
-https://docs.google.com/document/d/10pPuYIQyzvUHcUOLCjlP7qWjj2qGqdNFRzZIwPNPCWw/edit?usp=sharing
+# https://docs.google.com/document/d/10pPuYIQyzvUHcUOLCjlP7qWjj2qGqdNFRzZIwPNPCWw/edit?usp=sharing
```

Comment:
This change should be reverted back to 
`https://docs.google.com/document/d/10pPuYIQyzvUHcUOLCjlP7qWjj2qGqdNFRzZIwPNPCWw/edit?usp=sharing`
Changes to the header will create conflicts